apps/go-mod-validator/test/__snapshots__/deps.test.ts.snap

```diff
@@ -0,0 +1,99 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
This seems like a weird thing to snapshot. The result of the method should never change and this leaves it open to accidentally updating the snapshot.